code/modules/uplink/uplink_item.dm

```diff
@@ -633,6 +633,12 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	item = /obj/item/toy/carpplushie/dehy_carp
 	cost = 1
 
+/datum/uplink_item/stealthy_weapons/soap_clusterbang
+	name = "Slipocalypse Clusterbang"
+	desc = "A traditional clusterbang grenade with a payload consisting entirely of Syndicate soap. Useful in any scenario!"
+	item = /obj/item/weapon/grenade/clusterbuster/soap
+	cost = 2
+
```

Comment:
Well he said traitor in the title, so I thought I'd ask.

An entire area that nobody but the nuke team can move in would be very strong as well